stravalib/model.py

```diff
@@ -417,32 +639,44 @@ def is_authenticated_athlete(self):
         return self.is_authenticated
 
 
+# TODO: better description
```

Comment:
I wasn't sure exactly why this was being overwritten so wouldn't mind input on what the description here should be. there are a few instances of this where attrs are overwritten via a class like this.